pages/contacts-call/[id]/index.js

```diff
@@ -24,11 +27,6 @@ const StyledBigHeader = styled.h2`
   margin: 0;
 `;
 
-const StyledBigImage = styled(StyledImage)`
-  width: 9rem;
-  height: 9rem;
-  margin: 0;
-`;
 const StyledRedButton = styled.div`
```

Comment:
I think `StyledButton` as name can be enough for the same reason I described above for `GrayCirclePhotoPlaceholder`. 🤔

Reply:
changed